ush/templates/FV3LAM_wflow.xml

```diff
@@ -109,6 +120,11 @@ tasks; and the "FCST" type is used for the RUN_FCST_TN task.
 {# Double quotes are required inside the strftime! Expect an error from reading the template if using single quotes. #}
   <cycledef group="at_start">{{ cdate_first_cycl.strftime("%M %H %d %m %Y *") }}</cycledef>
   <cycledef group="forecast">{{ cdate_first_cycl.strftime("%Y%m%d%H%M") }} {{ cdate_last_cycl.strftime("%Y%m%d%H%M") }} {{ cycl_freq }}</cycledef>
+{%- if do_dacycle %}
+  <cycledef group="dacycle">00 {{ da_cycle_interval_hrs }}-11/{{ da_cycle_interval_hrs }},{{ da_cycle_interval_hrs + 12 }}-23/{{ da_cycle_interval_hrs }} {{cdate_first_cycl.strftime("%d") }}-{{ cdate_last_cycl.strftime("%d") }} {{ cdate_first_cycl.strftime("%m") }} {{cdate_first_cycl.strftime("%Y") }} *</cycledef>
+{%- else %}
+  <cycledef group="dacycle">00 00 01 01 2100 *</cycledef> 
```

Comment:
I'm unclear what the dacycle group is here, can it not be identical to the forecast group for non cycled experiments?

Reply:
No sure I understand this question. Try to give some answers:
The forecast group is only for cold start cycles so we use dacycle for representation the warm (continue) cycles.
Two group have many same tasks but the recycle doesn't need make_ics and analysis task need different dependences.  
When we want to run non cycled experiments, just set dacycle to a long future time so we don't need to add all those if checks to each task.